### ISSUE TITLE
Additional families

### ISSUE THREAD

**Issue by noamross** [vdorie/stan4bart#10]
Hello, I was wondering if you expected to eventually support additional families such as Poisson, Gamma, and NegativeBinomial in this package or dbarts, or if you see a path for users to do so, such as making a`dbarts` interface that can be used in Stan code.

**Comment by vdorie** [comment]
Yes, I'm planning on generalizing the base dbarts model to allow for different likelihoods. It's slow going, as I don't have much time these days.

**Comment by pcinereus** [comment]
Has there been any progress on this?  I too would love to be able to use Binomial, Poisson and NegBin in particular.  Ta
